pkg/controller/common/api/v1/common.go

```diff
@@ -0,0 +1,158 @@
+package v1
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// RelativeObjectMap holds object related to given parent object.
```

Comment:
disclaimer: I understand this is refactoring of existing packages. However provided some nits & suggestions at some places.

Reply:
yes, thank you